tests/phpunit/testdata/plugins/test-plugin-plugin-readme-errors/readme.txt

```diff
@@ -0,0 +1 @@
+empty
```

Comment:
This isn't really suitable as the only "error" readme test candidate as it doesn't cater for any of the nuances in testing. For example, what about a test for the default content being present in the readme? What about a test for the stable tag or license being invalid? Right now, all this checks is that none of that information is not present, but not the validity _when_ it is present.

Reply:
Thanks @felixarntz I've updated this with more content for testing. I've also created other test plugins to cover all the warning messages that this check creates.